callback.go

```
@@ -28,11 +28,17 @@ type callback struct {
 	Longitude      int64   `json:"lng"`
 	SequenceNumber int64   `json:"seqNumber"`
 	Bidirectional  bool    `json:"ack"`
+	IsDuplicate    bool    `json:"duplicate"`
 }
 
 // parseCallback parses a http.Request and returns a base sigfox callback
 func parseCallback(r *http.Request, cb *callback) (int, error) {
-	defer r.Body.Close()
+	defer func() {
+		err := r.Body.Close()
+		if err != nil {
+			log.Printf("body close: %v", err)
```

I'm not sure about this. It's breaks with the idiom by which most libraries swallow closing errors. Also, logging to stdout is kinda a carnal sin for libraries!

---

True for libs, but, log.XXX logs to default logger to stderr.

https://github.com/golang/go/blob/master/src/log/log.go#L73